infinispan-client/src/main/java/io/quarkus/ts/openshift/infinispan/client/InfinispanFirstGreetingResource.java

```diff
@@ -23,7 +23,7 @@ public String hello() {
         return cache.get("hello");
     }
 
-    @POST
+    @PUT
     @Produces(MediaType.TEXT_PLAIN)
     public String updateHello(String text) {
         cache.put("hello","Hello World, Infinispan is up and changed!");
```

Comment:
With regards to my review, the change would be `counter++` instead of just `cache.put()`